scheds/rust/scx_bpfland/src/main.rs

```diff
@@ -562,7 +562,7 @@ impl<'a> Scheduler<'a> {
     }
 }
 
-impl<'a> Drop for Scheduler<'a> {
+impl Drop for Scheduler<'_> {
```

Comment:
This change could be applied pretty much to all the Rust schedulers. Maybe send a separate PR with this particular change targeting all schedulers, instead of just bpfland?